1_fetch.R

```diff
@@ -115,11 +115,21 @@ p1_targets_list <- list(
     iteration = "group"
   ),
 
-  # Map over groups of sites to download data   
+  # Map over groups of sites to download data.
+  # Note that because error = 'continue', {targets} will attempt to build all 
+  # of the "branches" represented in p1_site_counts_grouped, even if one branch 
+  # returns an error. This way, we will not need to re-build branches that have
+  # already run successfully. However, if a branch fails, {targets} will throw
+  # an error here because it cannot merge the individual branches into a single
+  # data frame. The error(s) associated with the failed branch will therefore 
```

Comment:
```suggestion
  # an error reading `could not load dependencies of [immediate downstream target]. invalid 
  # 'description' argument` because it cannot merge the individual branches and so did not  
  # complete the branching target. The error(s) associated with the failed branch will therefore 
```